src/libs/actions/PersonalDetails.js

```diff
@@ -155,7 +155,7 @@ function getFromReportParticipants(reports) {
         .value();
 
     if (participantEmails.length === 0) {
-        return;
+        return Promise.resolve([]);
```

Comment:
I think the `details` is an object so this should be an object instead of `[]`. Please check and update it accordingly.